### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yhan818**
Update README.md for detailed documentation on how to clone a copy for your GitHub and connect it to your readthedocs.org. This is import documentation for users to understand how to connect GitHub and readthedocs.org.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
 # redata-docs
-A repo that provides general documentation for software pertaining to ReDATA, the University of Arizona Research Data Repository
\ No newline at end of file
+This is a repository that provides general documentation for software pertaining to ReDATA, the University of Arizona Research Data Repository. It provides links to other Re DATA software packages in Github and readthedocs.org.
```
Comment (zoidy) [inline README.md]: Re DATA -> ReDATA

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
 # redata-docs
-A repo that provides general documentation for software pertaining to ReDATA, the University of Arizona Research Data Repository
\ No newline at end of file
+This is a repository that provides general documentation for software pertaining to ReDATA, the University of Arizona Research Data Repository. It provides links to other Re DATA software packages in Github and readthedocs.org. 
+
+
+The following is how to create a copy at your own environment at your github and readthedocs.org.
```
Comment (zoidy) [inline README.md]: Clarify why one might want to do this. E.g., "For testing purposes, the following...."

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
 # redata-docs
-A repo that provides general documentation for software pertaining to ReDATA, the University of Arizona Research Data Repository
\ No newline at end of file
+This is a repository that provides general documentation for software pertaining to ReDATA, the University of Arizona Research Data Repository. It provides links to other Re DATA software packages in Github and readthedocs.org. 
+
+
+The following is how to create a copy at your own environment at your github and readthedocs.org. 
+
+### 1. First create your desired directory and Git clone it to your local directory.
+%git clone git@github.com:UAL-RE/redata-docs.git
```
Comment (zoidy) [inline README.md]: For more readable docs, I recommend using Markdown code blocks E.g.,
`git clone git@github.com:UAL-RE/redata-docs.git`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,49 @@
...
+
+### 2. Remove the cloned local repository from its remote association.
+After you clone the git repository. Remote configuration is saved in ".git" directory. You can see "config" on ".git" for the remote and branch info. 
+so you need to run "git remote rm orgin" to remove remote association. You can see the changes in "config" (remote "origin" and branch "main" are removed). You can run "git remote -v" to list all remote repositories connected to the local repository.
+
+% git remote -v  
+
+% git remote rm origin 
+
+% git remote -v
```
Comment (zoidy) [inline README.md]: Please use code blocks. The % isn't needed
```
git remote -v 
git remote rm origin 
git remote -v 
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,49 @@
...
+% git remote rm origin 
+
+% git remote -v 
+
+
+
+### 3. Create a new github repository and connect it with your local repository.
+-- Go to your github to create a new repository. 
+ 
+-- Connect this local repository to the new repository, and then verify its remote.
```
Comment (zoidy) [inline README.md]: Please use Markdown formatting for cleaner docs
- Go to your github to create a new repository. 
- Connect this local repository to the new repository, and then verify its remote.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,49 @@
...
+
+% git remote add origin git@github.com:hanmgmt/test-docs2.git
+
+% git remote -v 
+ 
+-- push the local repository to the new repository.
+
+% git push branch -M main
+
+% git push -u origin main
```
Comment (zoidy) [inline README.md]: Please use Markdown formatting for cleaner docs

File: README.md [hunk 7]
```diff
@@ -1,2 +1,49 @@
...
+### 4. Read the Docs: import a repository
+-- Create an account in readthedocs.org 
+
+-- Import a Repository by click the "refresh" button. 
+
+-- Configure and build a version
+
+-- Wait a few minutes and see the compile message and "build completed" message. 
+
+-- Click the "View Docs" for the latest version (HTML) with the URL.
```
Comment (zoidy) [inline README.md]: Please use Markdown formatting for cleaner docs

**Review by zoidy — CHANGES_REQUESTED** [review]
Overall looks good. The only minor request relates to formatting. It makes the docs easier to read if we use Markdown to format code blocks and lists. See https://www.markdownguide.org/cheat-sheet/ for how to do it

**Comment by yhan818** [comment]
Thanks for the suggestions. I have updated the README.md. Please review again.

**Review by zoidy — APPROVED** [review]

**Pull request merged**